cmd/bb_browser/asset_registry.go

```diff
@@ -32,11 +31,12 @@ func registerAsset(path string, body []byte, contentType string) {
 }
 
 func init() {
-	registerAsset("/bootstrap.css", assets.BootstrapCSS, "text/css")
-	registerAsset("/bootstrap.js", assets.BootstrapJS, "application/javascript")
-	registerAsset("/favicon.png", assets.FaviconPNG, "image/png")
-	registerAsset("/jquery.js", assets.JQueryJS, "application/javascript")
-	registerAsset("/terminal.css", assets.TerminalCSS, "text/css")
+	registerAsset("/bootstrap.css", AssetsData["bootstrap.min.css"], "text/css")
```

Comment:
Could this somehow be written in a way that it explicitly fails on startup when certain files don’t exist in AssetsData?

Reply:
Would this version work for you?